src/utils.rs

```diff
@@ -10,3 +11,97 @@ pub fn read_file<P: AsRef<Path>>(file_name: P) -> Result<String> {
     file.read_to_string(&mut data)?;
     Ok(data)
 }
+
+#[derive(Debug)]
+pub struct CommandOutput {
+    pub stdout: String,
+    pub stderr: String,
+}
+
+impl PartialEq for CommandOutput {
+    fn eq(&self, other: &Self) -> bool {
+        self.stdout == other.stdout && self.stderr == other.stderr
+    }
+}
+
+/// Execute a command and return the output on stdout and stderr if sucessful
+pub fn exec_cmd(cmd: &str, args: &[&str]) -> Option<CommandOutput> {
+    log::debug!("Executing command '{:?}' with args '{:?}'", cmd, args);
```

Comment:
I think this would be better suited as a trace log as it explains how the code is executing.
```suggestion
    log::trace!("Executing command '{:?}' with args '{:?}'", cmd, args);
```